projects/ccf-eui/src/index.html

```diff
@@ -38,7 +38,7 @@
 </head>
 <body class="mat-typography">
   <ccf-eui
-    use-remote-api="false"
+    use-remote-api="true"
```

Comment:
Undo this change!

Reply:
This is supposed to be true initially. But I changed it in my previous PR and that got merged.